sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -492,7 +492,7 @@ class AstBuilder extends SqlBaseBaseVisitor[AnyRef] with Logging {
   }
 
   /**
-   * Add an [[Aggregate]] to a logical plan.
+   * Add an [[Aggregate]] or [[GroupingSets]] to a logical plan.
```

Comment:
Didn't check the javadoc and the change follows `Aggregate`. I'll see how to generate javadoc and check this out (as I've been meaning to do it anyway for some time). Thanks for inspiration!